app/Services/Patreon/Resources/Campaign.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Services\Patreon\Resources;
+
+class Campaign extends Resource
+{
+    /** @var int */
+    public $id;
+
+    /** @var string */
+    public $name;
+
+    /** @var int */
+    public $pledgedSum;
+
+    /** @var \App\Services\Patreon\Resources\ResourceCollection  */
+    public $rewards;
+
+    public function __construct(int $id, string $name, int $pledgedSum)
+    {
+        $this->id = $id;
+        $this->name = $name;
+        $this->pledgedSum = $pledgedSum;
+
+        $this->rewards = new ResourceCollection();
+    }
+
+    public function fillRewards(array $rewardRelationsData, ResourceCollection $rewards)
```

Comment:
I don't think we are going to display rewards. Remove all code regarding rewards.